Allow DevelopmentServer testing with internal and external plugins together (#23100)
trinodb/trino: testing/trino-server-dev/etc/plugin/README.md

> ### External Plugins Directory

Reviewer: ```suggestion
### External plugins
```

> External plugins can be added to this directory to test with `DevelopmentServer`. Each 

Reviewer: ```suggestion
You can add external plugins in a directory and test with `DevelopmentServer` class. Each 
```

> plugin should be added as its own subdirectory.

Reviewer: ```suggestion
plugin must be added as its own subdirectory and contain all necessary dependencies. 
```
